src/corelibs/U2Private/src/PluginSupportImpl.cpp

```diff
@@ -321,6 +322,11 @@ void PluginSupportImpl::updateSavedState(PluginRef *ref) {
 }
 
 QDir PluginSupportImpl::getDefaultPluginsDir() {
+#ifdef Q_OS_DARWIN
+    if (! QDir(AppContext::getWorkingDirectoryPath() + "/plugins").exists()) {
+        return QDir(BundleInfoMac::getPluginsSearchPath());
```

Comment:
getPluginSearchPath may return an invalid dir (empty string). In this case it is better to follow the default way but not to search in the random (current) dir.

Reply:
This issue is not addressed yet